src/main/java/com/kamycz3q/coursesystemspringboot/personalData/personalDataObject/PersonalData.java

```diff
@@ -0,0 +1,25 @@
+package com.kamycz3q.coursesystemspringboot.personalData.personalDataObject;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import lombok.Getter;
+import lombok.Setter;
+
+import java.sql.Timestamp;
+
+@Getter
+@Setter
+@Entity
+public class PersonalData {
+    @Id
+    @GeneratedValue(
+            strategy = GenerationType.AUTO
+    )
+    private Long id;
+    private String name, surname, email;
+    private Timestamp birthDate;
```

Comment:
Do określenia daty urodzenia lepiej posłużyć się typem `java.sql.Date`.
Prawdopodobnie nigdy w systemie nie będzie nam potrzebna godzina, minuta i sekunda czyjegoś urodzenia, a to zajmuje miejsce w bazie danych. Nie pamiętam ile to jest bajtów więcej, ale zakładając że nasz system ma przechowywać miliony rekordów, tylko przez tą jedną rzecz wydamy więcej kasy na przestrzeń dyskową, a zupełnie niepotrzebnie.

Reply:
Prawda, zmieniono na Date.